### PR TITLE
[webview_flutter] Use TextDirection.ltr for webview_android

### PR DESCRIPTION
**Pull request by Sunbreak**
Signed-off-by: Sunbreak <sunbreak.wang@gmail.com>

## Description

Change `TextDirection.rtl` to `TextDirection.ltr` as default value in `webview_android.dart`

## Related Issues

https://github.com/flutter/flutter/issues/66208

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/webview_flutter/lib/src/webview_android.dart [hunk 1]
```diff
@@ -50,7 +50,7 @@ class AndroidWebView implements WebViewPlatform {
         // WebView content is not affected by the Android view's layout direction,
         // we explicitly set it here so that the widget doesn't require an ambient
         // directionality.
-        layoutDirection: TextDirection.rtl,
+        layoutDirection: TextDirection.ltr,
```
Comment (cyanglaz) [inline packages/webview_flutter/lib/src/webview_android.dart:53]: Looks like this doesn't take "Directionality.of(context)" into account. See https://github.com/flutter/flutter/issues/66208#issuecomment-738247198

**Review by cyanglaz — COMMENTED** [review]
Thanks for the PR. We will need to update the version and CHANGELOG before landing this.

**Comment by Sunbreak** [comment]
@cyanglaz Could you take another look?

**Comment by stuartmorgan** [comment]
Is this actually still necessary to change in the new version of the code? What's the case where this fallback is used?

My understanding is that Flutter generally doesn't do direction fallbacks to avoid being wrong half the time, so if this is still happening this is probably not the right fix.

**Comment by stuartmorgan** [comment]
Thank you for your contribution. I'm going to close this PR for now since there are outstanding comments, just to get this off our PR review queue. Please don't hesitate to submit a new PR if you have the time to address the question above. Thanks!

**Pull request closed without merging**